cfgov/templates/wagtailadmin/table_input.html

```diff
@@ -44,7 +44,7 @@
                 <input type="text"
                        id="{{ attrs.id }}-handsontable-heading-icon"
                        name="handsontable-heading-icon">
-                <span class="help">Input the name of an icon to appear to the left of the heading. E.g., approved, help-round, etc. See full list of icons</span>
+                <span class="help">Input the name of an icon to appear to the left of the heading. E.g., approved, help-round, etc. <a href="https://cfpb.github.io/capital-framework/'components/cf-icons/#icons">See full list of icons</span>
```

Comment:
@jimmynotjim OMG I forgot the closing `</a>` 🤦‍♂️

Reply:
Whoops